src/new/sendTransaction.js

```diff
@@ -172,15 +276,72 @@ class SendTransaction extends EventEmitter {
     }).catch(() => {
       this.updateOutputSelected(false);
       this.emit('send-error', this.unexpectedPushTxError);
-    });;
+    });
   }
 
   /**
-   * Start object (submit job)
+   * Run sendTransaction from mining, i.e. expects this.transaction to be prepared and signed
+   * then it will mine and push tx
+   *
+   * 'until' parameter can be 'mine-tx', in order to only mine the transaction without propagating
+   *
+   * @memberof SendTransaction
+   * @inner
+   */
+  runFromMining(until = null) {
```

Comment:
Shouldn't it return a promise that will be resolved or rejected accordingly?

Reply:
Done here https://github.com/HathorNetwork/hathor-wallet-lib/pull/221/commits/43cdd56eebf3e443132890aaddd495d24aaf24f5

Reply:
That's great! Thanks!